material_components_ios.bzl

```diff
@@ -272,27 +272,42 @@ def mdc_unit_test_suite(
     minimum_os_version = IOS_MINIMUM_OS,
     visibility = ["//visibility:private"],
     size = "medium",
+    use_autobot_extension_runner = True,
     **kwargs):
-    """Declare a MDC unit_test_suite and a unit_test_extension using the ios_runners matrix."""
+    """Declare a MDC unit_test_suite and a unit_test_extension using the ios_runners matrix.
+
+    Args:
+        name: The name of the target.
+        deps: The dependencies of the target.
+        minimum_os_version: The minimum iOS version supported by the target.
+        visibility: The visibility of the package.
+        size: The size of the test.
+        use_autobot_extension_runner: Indicates whether autobot extension runner should be used.
```

Comment:
Please describe what an autobot is here.

Reply:
Addressed in 33af2a9473c239e6d7c70c4117c68ae576ae7a5c. I didn't put iOS 13 keyword there because it could be used for a broader scope.